### PR TITLE
Conditional conversion operator

### PR DESCRIPTION
**Pull request by sissonen**
Mapping operator for converting original values. Details in doc/03_Configuration/06_Mapping_Configuration/02_Transformation_Pipeline.md

### CODE REVIEW

**Comment by sissonen** [comment]
recheck

**Comment by fashxp** [comment]
you have to type the phase `I have read the CLA Document  ... ` as comment as suggested in the comment above to make CLA happy. 
Thank you!

File: src/Mapping/Operator/Simple/ConditionalConversion.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    public function process($inputData, bool $dryRun = false)
+    {
+        $returnScalar = false;
+        if (!is_array($inputData)) {
+            $returnScalar = true;
+            $inputData = [$inputData];
+        }
+
+        \Pimcore\Log\Simple::log("importerlog", $this->original . " " . $this->converted ." ".print_r($inputData, true));
```
Comment (fashxp) [inline src/Mapping/Operator/Simple/ConditionalConversion.php]: can we remove that line? it will fill up the disk over time...

**Review by fashxp — CHANGES_REQUESTED** [review]
nice addition

**Comment by sissonen** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by fashxp — APPROVED** [review]

**Comment by fashxp** [comment]
thx very much!

**Pull request merged**